llvm/utils/update_test_prefix.py

```diff
@@ -17,24 +17,24 @@ def remove_prefix(i, d=0):
             p = r.group(1)
             s = re.sub("=" + p + ",", "=", s)
             s = re.sub("," + p + "([, \n])", "\\1", s)
-            s = re.sub("\s+-?-check-prefix=" + p + "([ \n])", "\\1", s)
+            s = re.sub("\\s+-?-check-prefix=" + p + "([ \n])", "\\1", s)
```

Comment:
I'd prefer to use raw strings here as well. `\n` is both a python and regex escape sequence, and there's no need to use the python one here, if it makes things less similar to other places.